Core/TimberApi/AssetSystem/AssetFolder.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using TimberApi.AssetSystem.Exceptions;
+using TimberApi.Common.Helpers;
+using TimberApi.SceneSystem;
+
+namespace TimberApi.AssetSystem
+{
+    internal class AssetFolder
+    {
+        private static readonly string[] AllowedExtensions = {"", ".bundle", ".asset"};
```

Comment:
is there a reason we are restricting the asset bundle extension naming to these three?

Reply:
haven't found any other extensions used for them. I saw you can do customs ones. But with this you could also add jsons inside or readme's. Maybe in the future plain .jpg files that can be imported for sprites in game